src/collections/news/2023-05-17-Baby-Bliss-Bot-AI-Techniques.md

```diff
@@ -106,8 +107,7 @@ recognize new objects not previously included in the datasets.
 The BBB Project has started experimenting with CLIP models in order to explore
 how the use of multimodal (images and textual) data training strategies can be
 applied for a better understanding of the internal composition of Bliss symbols
-and combined words, and eventually provide avenues for the generation of new
-symbols.
+and combined words.
```

Comment:
This changes what @cindyli wrote since she contributed the StyleGAN3 content.  It's not clear why the clause about generating new symbols idea has been removed.  What do you think @cindyli?

Reply:
Ignore me, @agamba.  I misinterpretted the diff view and this is part of the CLIP section, not the StyleGAN3 section.  If you want to remove the generation clause, it's fine.